daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -56,7 +56,7 @@ func init() {
 // New creates new JSONFileLogger which writes to filename passed in
 // on given context.
 func New(ctx logger.Context) (logger.Logger, error) {
-	log, err := os.OpenFile(ctx.LogPath, os.O_RDWR|os.O_APPEND|os.O_CREATE, 0600)
+	log, err := os.OpenFile(ctx.LogPath, os.O_RDWR|os.O_APPEND|os.O_CREATE, 0666)
```

Comment:
yes `0640` is looks better. I was still don't confident which is better permission. so following previous permission. 
Anyway can I set option to write log file in other directory? I couldn't any option. if don't is it needed?(new implementation needed?)